docs/building-on-lisk/web3-app-development.mdx

```diff
@@ -25,7 +25,7 @@ import TabItem from '@theme/TabItem';
 
 On this page, you will learn what Web3 applications are, how they differ from Web 2.0 apps, and how to get started with Web3 app development on Lisk.
 
-## What are Web3 applications?
+## What are Web3 Applications?
```

Comment:
I would wait for confirmation from marketing before choosing a capitalization style.